app/assets/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.GraduatedPrice.js

```diff
@@ -62,24 +61,35 @@
     },
 
     advocateCategory: function() {
-      return $('input:radio[name="claim[advocate_category]"]:checked').val();
+      return this.getVal('input:radio[name="claim[advocate_category]"]:checked');
+    },
+
+    feeTypeId: function(context) {
+      return this.getVal(context, '.js-fee-type');
     },
 
-    feeTypeId: function() {
-      return $('.fx-fee-group').find('.js-fee-type').val();
+    ppe: function(context) {
+      return this.getVal(context, 'input.js-fee-calculator-ppe');
     },
 
-    ppe: function() {
-      return $('.fx-fee-group').find('input.js-fee-calculator-ppe').val();
+    pw: function(context) {
+      return this.getVal(context, 'input.js-fee-calculator-pw');
     },
 
-    days: function() {
-      return $('.fx-fee-group').find('input.js-fee-calculator-days:visible').val();
+    days: function(context) {
+      return this.getVal(context, 'input.js-fee-calculator-days:visible');
+    },
+
+    getVal: function(context, locatorString){
```

Comment:
i'd rename `locatorString` --> `selector` as is a common name for such things

Reply:
👍